rom/dev/tests/test_image_validation.rs

```diff
@@ -36,6 +36,17 @@ const VENDOR_PUB_KEY_DIGEST_MISMATCH: u32 = 0x0B000005;
 const VENDOR_ECC_PUB_KEY_REVOKED: u32 = 0x0B000009;
 const VENDOR_ECC_PUB_KEY_INDEX_MISMATCH: u32 = 0x0B00000D;
 const VENDOR_ECC_PUB_KEY_INDEX_OUT_OF_BOUNDS: u32 = 0x0B000008;
+const FMC_LOAD_ADDR_INVALID: u32 = 0x0B000021;
+const FMC_ENTRY_POINT_INVALID: u32 = 0x0B000022;
+const FMC_SVN_GREATER_THAN_MAX_SUPPORTED: u32 = 0x0B000023;
+const FMC_SVN_LESS_THAN_MIN_SUPPORTED: u32 = 0x0B000024;
+const RUNTIME_LOAD_ADDR_INVALID: u32 = 0x0B000025;
+const RUNTIME_ENTRY_POINT_INVALID: u32 = 0x0B000026;
+const RUNTIME_SVN_GREATER_THAN_MAX_SUPPORTED: u32 = 0x0B000027;
+const RUNTIME_SVN_LESS_THAN_MIN_SUPPORTED: u32 = 0x0B000028;
+
+const ICCM_START_ADDR: u32 = 0x40000000;
```

Comment:
These probably need to be defined somewhere centrally too (another TODO?)

Reply:
Agreed. Currently I have them under a TODO.